bedrock/firefox/views.py

```diff
@@ -620,7 +620,55 @@ def hello_screen_sharing(version):
     return version >= Version('38.0.5')
 
 
-class HelloStartView(LatestFxView):
+class HelloFxView(TemplateView):
```

Comment:
I see your note about a separate class for this to avoid changes, but we can handle that if/when it comes up I think.